ui/page/receive_page.go

```diff
@@ -137,6 +138,16 @@ func (pg *ReceivePage) OnNavigatedTo() {
 	pg.selector.ListenForTxNotifications(pg.ctx)
 	pg.selector.SelectFirstWalletValidAccount(nil) // Want to reset the user's selection everytime this page appears?
 	// might be better to track the last selection in a variable and reselect it.
+	selectedWallet := pg.multiWallet.WalletWithID(pg.selector.SelectedAccount().WalletID)
+	currentAddress, err := selectedWallet.CurrentAddress(pg.selector.SelectedAccount().Number)
+	if err != nil {
+		log.Errorf("Error getting current address: %v", err)
+		pg.Toast.NotifyError(fmt.Sprintf("Error getting current address: %v", err))
+	} else {
+		pg.currentAddress = currentAddress
+	}
```

Comment:
In regards displaying the error to the user, I'm not sure it's required, as this is not a user generated action and can be confusing, seems better to just log the response only

errors are usually useful when the user has tried to perform an action, and they are told the result of that action, or directed to what needs to be done, telling the user there's an error getting the current address is confusing as they still see an address and are left perplexed

```suggestion
	if err != nil {
		log.Errorf("Error getting current address: %v", err)
	} else {
		pg.currentAddress = currentAddress
	}
```

Reply:
@Sirmorrison @itswisdomagain thoughts?

Reply:
My take on errors is basically different. it provides the user with facts that an error has occurred while trying out a process rather than leave them in limbo hoping for result. 

looking at the code, it is an `if` `else` scenario meaning this should happen or this and as such we are expecting 2 outcomes. when one fails, user need to be notified so they dont keep waiting for the output of the other. 

displaying the error would not really matter if it does not have impact on the UX, however in this regard it does.

Address will not the generated and the user will not know why and as such, nothing will be displayed on the UI for the user to work with.

https://github.com/planetdecred/godcr/blob/f31dfb8d0d23bad0aee029985177b3e9a4624092/ui/page/receive_page.go#L208

Reply:
alright good point